app/assets/stylesheets/notes.scss

```diff
@@ -19,50 +19,26 @@ td{
   color: black;
 }
 
-.new {
-  font-size: 16px;
-  margin-left: 20%;
-  padding: 20px;
-}
-
 #search-box{
   padding: 16px;
 }
 
 .wrapper {
-  width: 100%;
-  max-width: 50rem;
-  margin-left: 20%;
-  padding: 20px;
-}
-
-.searchBar {
-  width: 100%;
   display: flex;
-  flex-direction: row;
-  align-items: center;
+  margin-left: 10%;
+  margin-right: 10%;
+  padding-top: 20px;
+  padding-bottom: 20px;
 }
 
-#searchQueryInput {
-  width: 100%;
-  height: 2.8rem;
+#txtSearch {
```

Comment:
Prefer using complete words. In this case `textSearch` would be better.

Consider `searchInput`, since it is an input field that is being used for search.

Reply:
Right, I will replace with `searchInput`